### PR TITLE
Bump awb to 2.1.0a0

### PR DESCRIPTION
**Pull request by unkcpz**
Fix the failed test by set the code dropdown from code uuid.

### CODE REVIEW

File: tests/conftest.py [hunk 1]
```diff
@@ -301,15 +301,10 @@ def app(pw_code, dos_code, projwfc_code):
...
-        app.submit_step.pw_code.code_select_dropdown.options[pw_code.full_label]
-    )
-    app.submit_step.codes["dos"].value = app.submit_step.codes[
-        "dos"
-    ].code_select_dropdown.options[dos_code.full_label]
-    app.submit_step.codes["projwfc"].value = app.submit_step.codes[
-        "projwfc"
-    ].code_select_dropdown.options[projwfc_code.full_label]
+
+    app.submit_step.pw_code.value = pw_code.uuid
```
Comment (unkcpz) [inline tests/conftest.py:305]: The fixture is a code node, there is no need to get the uuid from the option.

**Comment by unkcpz** [comment]
The image build test failed because the exception below:

```
++ python -m aiidalab_qe install-qe                                                                                                                                                                                                           
Traceback (most recent call last):                                                                                     
  File "/opt/conda/lib/python3.9/runpy.py", line 197, in _run_module_as_main                                           
    return _run_code(code, main_globals, None,                                                                         
  File "/opt/conda/lib/python3.9/runpy.py", line 87, in _run_code                                                      
    exec(code, run_globals)                                                                                            
  File "/opt/conda/lib/python3.9/site-packages/aiidalab_qe/__main__.py", line 6, in <module>                           
    from aiidalab_qe.app.submission.sssp import install as setup_sssp                                                  
  File "/opt/conda/lib/python3.9/site-packages/aiidalab_qe/app/__init__.py", line 3, in <module>                       
    from .main import App                                                                                              
  File "/opt/conda/lib/python3.9/site-packages/aiidalab_qe/app/main.py", line 14, in <module>                                                                                                                                                 
    from aiidalab_qe.app.submission import SubmitQeAppWorkChainStep                                                    
  File "/opt/conda/lib/python3.9/site-packages/aiidalab_qe/app/submission/__init__.py", line 21, in <module>                                                                                                                                  
    from aiidalab_qe.workflows import QeAppWorkChain                                                                   
  File "/opt/conda/lib/python3.9/site-packages/aiidalab_qe/workflows/__init__.py", line 42, in <module>                
    plugin_entries = get_entry_items("aiidalab_qe.properties", "workchain")                                            
  File "/opt/conda/lib/python3.9/site-packages/aiidalab_qe/workflows/__init__.py", line 34, in get_entry_items         
    entries = get_entries(entry_point_name)                                                                                                                                                                                                   
  File "/opt/conda/lib/python3.9/site-packages/aiidalab_qe/workflows/__init__.py", line 27, in get_entries                                                                                                                                    
    entries[entry_point.name] = entry_point.load()                                                                                                                                                                                            
  File "/opt/conda/lib/python3.9/site-packages/importlib_metadata/__init__.py", line 207, in load                      
    module = import_module(match.group('module'))                                                                      
  File "/opt/conda/lib/python3.9/importlib/__init__.py", line 127, in import_module                                    
    return _bootstrap._gcd_import(name[level:], package, level)                                                        
  File "/opt/conda/lib/python3.9/site-packages/aiidalab_qe/plugins/pdos/__init__.py", line 15, in <module>             
    dos_code = ComputationalResourcesWidget(                                                                                                                                                                                                  
  File "/opt/conda/lib/python3.9/site-packages/aiidalab_widgets_base/computational_resources.py", line 112, in __init__                                                                                                                       
    self.resource_setup = _ResourceSetupBaseWidget(                                                                    
  File "/opt/conda/lib/python3.9/site-packages/aiidalab_widgets_base/computational_resources.py", line 1632, in __init__                                                                                                                      
    self.aiida_code_setup = AiidaCodeSetup()                                                                                                                                                                                                  
  File "/opt/conda/lib/python3.9/site-packages/aiidalab_widgets_base/computational_resources.py", line 1085, in __init__                                                                                                                      
    self.computer = ComputerDropdownWidget()                                                                           
  File "/opt/conda/lib/python3.9/site-packages/aiidalab_widgets_base/computational_resources.py", line 1328, in __init__                                                                                                                      
    self.refresh()                                                                                                     
  File "/opt/conda/lib/python3.9/site-packages/aiidalab_widgets_base/computational_resources.py", line 1348, in refresh                                                                                                                       
    self._dropdown.options = self._get_computers()                                                                     
  File "/opt/conda/lib/python3.9/site-packages/aiidalab_widgets_base/computational_resources.py", line 1335, in _get_computers                                                                                                                
    user = orm.User.collection.get_default()                                                                           
  File "/opt/conda/lib/python3.9/site-packages/aiida/common/lang.py", line 99, in __get__                              
    return self.getter(owner)                                                                                                                                                                                                                 
  File "/opt/conda/lib/python3.9/site-packages/aiida/orm/entities.py", line 184, in collection                         
    return cls._CLS_COLLECTION.get_cached(cls, get_manager().get_profile_storage())                                                                                                                                                           
  File "/opt/conda/lib/python3.9/site-packages/aiida/manage/manager.py", line 242, in get_profile_storage              
    raise ConfigurationError(                                                                                          
aiida.common.exceptions.ConfigurationError: Could not determine the current profile. Consider loading a profile using `aiida.load_profile()`. 
```

I think it is caused by the `load_profile()` is not run in the package scope https://github.com/aiidalab/aiidalab-widgets-base/pull/481.

**Comment by unkcpz** [comment]
Okay, problem solved by 1) move the pseudo install modules out to `aiidalab_qe/common`. 2) run `load_profile` explicitly in `install_pseudo` and `install_qe` functions which are called from the command (`python -m aiidalab-qe install-qe`) to pre-install the pseudo and QE.

**Comment by unkcpz** [comment]
@superstar54 please have a look when you find time next week, this blocks the https://github.com/aiidalab/aiidalab-qe/pull/519

**Review by superstar54 — APPROVED** [review]
thanks. LGTM!

**Pull request merged**